tests/e2e/suites/register-basic.lc.test.ts

```diff
@@ -14,11 +14,13 @@ import { setupAndGo } from '../utilities/setup-and-go';
 describe('Test basic registration flow', () => {
   ['chromium', 'webkit', 'firefox'].forEach((browserType) => {
     const un = v4();
+    const pw = 'ieH034K&-zlwqh3V_';
```

Comment:
@spetrov Are you adding this because of that hardcoded check in the mock server?

Reply:
@cerebrl Yes, the mock server is expecting this value. Without adding the value explicitly here, it is taking the password from /tests/e2e/env.config.ts and the test against mock server fails...

Reply:
Would you mind changing the mock server to take the password from the configuration, rather than having to make the change here? That way, we can keep the configuration for all of this in the same place.

I just pulled and ran your branch locally, and the following change in `routes.auth.mjs` works as expected: https://share.getcloudapp.com/o0umYKZo.

Other than this, the PR looks good.

Reply:
Yes, this is so much better than it was! 👍 
Fixed it as suggested!